esp/bindings/http/platform/httpservice.cpp

```diff
@@ -402,29 +411,30 @@ int CEspHttpServer::processRequest()
         // so maximize the amount of request processing that can be traced. Specifically, user
         // authentication and authorization may generate trace output.
         bool wantTracing = true;
```

Comment:
minor: Should be initialised to queryTraceManager().isTracingEnabled()
otherwise it will be true if it is an unhandled get method even if that function returns false.